functests/manifests/template-with-rules-incorrect-just-warning.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: fedora-desktop-small-with-rules-incorrect
+  annotations:
+    openshift.io/display-name: "Fedora 23+ VM"
+    description: >-
+      This template can be used to create a VM suitable for
+      Fedora 23 and newer.
+      The template assumes that a PVC is available which is providing the
+      necessary Fedora disk image.
+
+      Recommended disk image (needs to be converted to raw)
+      https://download.fedoraproject.org/pub/fedora/linux/releases/28/Cloud/x86_64/images/Fedora-Cloud-Base-28-1.1.x86_64.qcow2
+    tags: "kubevirt,virtualmachine,fedora,rhel"
+
+    iconClass: "icon-fedora"
+    openshift.io/provider-display-name: "KubeVirt"
+    openshift.io/documentation-url: "https://github.com/kubevirt/common-templates"
+    openshift.io/support-url: "https://github.com/kubevirt/common-templates/issues"
+    template.openshift.io/bindable: "false"
+
+    template.kubevirt.io/version: v1alpha1
+    defaults.template.kubevirt.io/disk: rootdisk
+    template.kubevirt.io/editable: |
+      /objects[0].spec.template.spec.domain.cpu.sockets
+      /objects[0].spec.template.spec.domain.cpu.cores
+      /objects[0].spec.template.spec.domain.cpu.threads
+      /objects[0].spec.template.spec.domain.resources.requests.memory
+      /objects[0].spec.template.spec.domain.devices.disks
+      /objects[0].spec.template.spec.volumes
+      /objects[0].spec.template.spec.networks
+
+    name.os.template.kubevirt.io/fedora26: Fedora 26
+    name.os.template.kubevirt.io/fedora27: Fedora 27
+    name.os.template.kubevirt.io/fedora28: Fedora 28
+    validations: |
+      [
+        {
+          "name": "EnoughMemory",
+          "path": "jsonpath::.spec.domain.resources.requests.memory",
+          "message": "Memory size not within range",
+          "rule": "integer",
+          "min": 67108864,
+          "max": 536870912,
+          "justWarning": true
```

Comment:
we need to update the spec here: https://github.com/kubevirt/common-templates/blob/master/templates/VALIDATION.md - It's totally fine doing it in a later patch.

Reply:
I know, it will follow in another patch

Reply:
PR is here: https://github.com/kubevirt/common-templates/pull/77